libs/utils/lana.js

```diff
@@ -83,7 +85,7 @@
       queryParams.push('tags=' + encodeURI(o.tags));
     }
 
-    if (w.lana.debug || w.lana.localhost) console.log('LANA Msg: ', msg, '\nOpts:', o);
+    if (!isProdDomain || w.lana.debug || w.lana.localhost) console.log('LANA Msg: ', msg, '\nOpts:', o);
```

Comment:
shouldn't it be `window.lana?.log` instead of `console.log`?

Reply:
This is inside the `lana.log` function.  When lana log is called on a non-prod domain, the message is output to the console